Refit/refit.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
     <metadata>
         <id>refit</id>
-        <version>2.4.1</version>
+        <version>2.4.2</version>
```

Comment:
This is a breaking change to Windows 8 Store and Windows 8 Phone users - bumping the patch is no the right digit to increment.